### ISSUE TITLE
Allow enabling/disabling the 'multi-header' attribute on a lambda target group

### ISSUE THREAD

**Issue by paultyng** [terraform-providers/terraform-provider-aws#7637]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS has the ability to enable this attribute in the console.  It appears this is also possible with the following API calls:

https://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_ModifyTargetGroupAttributes.html

https://docs.aws.amazon.com/sdk-for-go/api/service/elbv2/#ELBV2.ModifyTargetGroupAttributes

### New or Affected Resource(s)

* `aws_lb_target_group`

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by bflad** [comment]
Pull request submitted: #7648

**Comment by bflad** [comment]
Support for this new attribute has been merged and will release with version 2.1.0 of the Terraform AWS Provider, likely middle of next week.

**Comment by bflad** [comment]
This has been released in [version 2.1.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#210-march-07-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
